<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CpuWidget</class>
 <widget class="QWidget" name="CpuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>563</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Monospace</family>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
       <item>
        <widget class="QTabWidget" name="tabWidgetRegFunc">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>350</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabRegisters">
          <attribute name="title">
           <string>Registers</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="RegisterWidget" name="registerWidget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>320</width>
               <height>300</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>595</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Monospace</family>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabFunctions">
          <attribute name="title">
           <string>Functions</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QListWidget" name="listFunctions">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="btnRefreshFunctions">
               <property name="text">
                <string>Refresh</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtFuncSearch">
               <property name="placeholderText">
                <string>Filter</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Memory Search</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="5" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>End</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1" colspan="3">
              <widget class="QLineEdit" name="txtSearchStart">
               <property name="text">
                <string>0x00</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Value</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="txtSearchValue"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1" colspan="3">
              <widget class="QLineEdit" name="txtSearchEnd">
               <property name="text">
                <string>0x2000000</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Type</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="cmbSearchType">
               <item>
                <property name="text">
                 <string>1 Byte (8 bits)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2 Bytes (16 bits)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4 Bytes (32 bits)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8 Bytes (64 bits)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Float</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Double</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>String</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Hex</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QCheckBox" name="chkSearchHex">
               <property name="text">
                <string/>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="2" colspan="2">
              <widget class="QPushButton" name="btnSearch">
               <property name="text">
                <string>Search</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListWidget" name="listSearchResults"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="DisassemblyWidget" name="disassemblyWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>350</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>595</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Monospace</family>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>215</height>
        </size>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_memory">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="title">
         <string>Memory</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="MemoryViewWidget" name="memoryviewWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>145</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Monospace</family>
            </font>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_breakpoints">
        <attribute name="title">
         <string>Breakpoints</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableView" name="breakpointList">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="gridStyle">
            <enum>Qt::NoPen</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_threads">
        <attribute name="title">
         <string>Threads</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableView" name="threadList">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="gridStyle">
            <enum>Qt::NoPen</enum>
           </property>
           <property name="cornerButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_callstack">
        <attribute name="title">
         <string>Active Call Stack</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableView" name="stackList">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="gridStyle">
            <enum>Qt::NoPen</enum>
           </property>
           <property name="cornerButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisassemblyWidget</class>
   <extends>QWidget</extends>
   <header>pcsx2-qt/Debugger/DisassemblyWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RegisterWidget</class>
   <extends>QWidget</extends>
   <header>pcsx2-qt/Debugger/RegisterWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MemoryViewWidget</class>
   <extends>QWidget</extends>
   <header>pcsx2-qt/Debugger/MemoryViewWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
